src/services/ubi/community.ts

```diff
@@ -634,6 +636,15 @@ export default class CommunityService {
                         },
                     ],
                 },
+                {
+                    model: this.ubiBeneficiaryRegistry,
+                    as: 'beneficiaryRegistry',
+                    attributes: ['address'],
+                    required: false,
+                    where: {
+                        activity: UbiBeneficiaryRegistryType.add,
+                    },
+                },
```

Comment:
This is one of those situations where I would not use the include. The reason is this https://askinglot.com/can-a-left-join-create-duplicates. In fact, it also happens with the thumbnails (include above), but if you look close, there's a `separate: true`. "If true, runs a separate query to fetch the associated instances, only supported for hasMany associations" from https://sequelize.org/master/class/lib/model.js~Model.html#static-method-findAll (params details). With the "separate" it runs a subquery, which reduces de amount of process needed on the db but also on sequelize to properly generate the json object.

In this case, what we want is actually to count how many beneficiaries a manager has added. So I guess, a separate query `count` instead, is better.

Reply:
yes.. when I was doing this I thought about the `include`.
But, by `do a separate query` do you mean to add this `separate: true` or write a new query inside the `map` below to count the beneficiaries added by each manager?

Reply:
After reading your comment and thinking, I'd go with a separate query. Instead of one by manager, it can be done with a `select manager_address, count(added_addresses) from ubi_beneficiary_registry where manager_address in (the_list_goes_here) group by manager_address` (pseudo-code-query). What do you think?

Reply:
right, but make this query is the same that `separate: true` generate, isn't it?
if I use separate option, sequelize make this query and organize it with the JSON.

Reply:
yes, but it returns the UbiBeneficiaryRegistry and we don't need that for anything, we just need a number. Imagine a community with 1500 beneficiaries. It will basically fetch 1500 results from UbiBeneficiaryRegistry just to count them.

It might be possible though to make this `count` and `group` within the include, using the `separate: true` and that would simplify the code. I've never done that.

Reply:
This https://stackoverflow.com/a/63621045/3348623 looks nice, the problem is that you need to use `raw: true`, otherwise the `attributes:[]` still return the primary key, which is the entry id, if I'm not mistaken.

It's worth look at the underlying queries being made, and write some code documentation since this is getting a big complex.

Reply:
I decided to make a separate query, following your first recommendation. I tried to make this `count` with the `include` clause but didn't work well when you have others includes in this same query...